Drop-In/src/main/java/com/braintreepayments/api/DropInResult.java

```diff
@@ -55,18 +51,18 @@ void setDeviceData(@Nullable String deviceData) {
         this.deviceData = deviceData;
     }
 
-    void setPaymentMethodType(@Nullable DropInPaymentMethodType paymentMethodType) {
+    void setPaymentMethodType(DropInPaymentMethod paymentMethodType) {
         this.paymentMethodType = paymentMethodType;
     }
 
     /**
-     * @return The previously used {@link DropInPaymentMethodType} or {@code null} if there was no
-     * previous payment method. If the type is {@link DropInPaymentMethodType#GOOGLE_PAY} the Google
+     * @return The previously used {@link DropInPaymentMethod} or {@code null} if there was no
+     * previous payment method. If the type is {@link DropInPaymentMethod#GOOGLE_PAY} the Android
```

Comment:
```suggestion
     * previous payment method. If the type is {@link DropInPaymentMethod#GOOGLE_PAY} the Google
```

noticed one other change